test/interpolation_tests.jl

```diff
@@ -75,14 +75,6 @@ end
     @test A(3.0) == 2.0
     @test A(4.0) == 3.0
 
-    u = [0.0, NaN, 2.0, 3.0]
```

Comment:
Why was this removed?

Reply:
Unless I am missing something very subtle, this block of tests is identical to the previous one

Reply:
Ah yes, didn't see that.